src/utils/preprocessing/preprocessing.js

```diff
@@ -13,10 +15,17 @@ function tokenize(sentence) {
   return sentence.split(" ");
 }
 
+function isExpression(word) {
+  return word.includes(" ");
+}
+
 function isWordInSentence(word, sentence) {
```

Comment:
should also be named `isTextInSentence`? Given that we agreed that it can be either a word or an expression?